custom_components/mail_and_packages/const.py

```diff
@@ -733,6 +756,43 @@
         icon="mdi:package-variant-closed",
         key="dhl_parcel_nl_packages",
     ),
+    # Post NL
+    "post_nl_processing": SensorEntityDescription(
+        name="Post NL Processing",
+        native_unit_of_measurement="package(s)",
+        icon="mdi:dolly",
+        key="post_nl_preparing",
+    ),
+    "post_nl_collection": SensorEntityDescription(
+        name="Post NL Ready for Collection",
+        native_unit_of_measurement="package(s)",
+        icon="mdi:store-clock",
+        key="post_nl_ready_for_collection",
```

Comment:
```suggestion
        key="post_nl_collection",
```
They key needs to match the sensor.